src/protocolsupportpocketstuff/api/PocketStuffAPI.java

```diff
@@ -1,17 +1,27 @@
 package protocolsupportpocketstuff.api;
 
+import protocolsupportpocketstuff.util.resourcepacks.ResourcePackManager;
+
 import java.math.BigInteger;
 
 public class PocketStuffAPI {
 
 	private static final BigInteger apiversion = BigInteger.valueOf(4);
-	
+	private static ResourcePackManager resourcePackManager;
```

Comment:
Make this in a different wrapper class inside the api -> resourcepacks

Reply:
Like this: https://github.com/ProtocolSupport/ProtocolSupport/blob/mcpenew/src/protocolsupport/api/unsafe/peskins/PESkinsProviderSPI.java